src/views/routing/sidebar/body/SidebarBody.web.js

```diff
@@ -23,9 +23,18 @@ const Sidebar = ({
   headerImageHeight = 200,
   side = 'left',
   backdropProps,
+  sortDirection,
+  sortBy,
 }) => {
+  const sortedItems = [...items];
+
+  /* If both a sort direction and sort by are provided then sort the items */
+  if ( sortDirection && sortBy ) {
+    sortDirection === 'desc' ? sortedItems.sort(( a, b ) => b[sortBy] > a[sortBy] ? 1 : -1 ) : sortedItems.sort(( a, b ) => a[sortBy] > b[sortBy] ? 1 : -1 );
+  }
+
```

Comment:
this block of code is being duplicated in both the web and native files, is there a way to move it into a parent so it it only used once?